### PR TITLE
Increase service timeout to 120s, add execution state UNKNOWN

### PR DESCRIPTION
**Pull request by rikroe**
## Proposed change

This PR is trying to cope with some longer standing issues regarding service execution:
- Increase default timeout to 120 seconds (as per https://github.com/bimmerconnected/bimmer_connected/issues/190)
- Add an `UNKOWN` execution state if we don't get a valid API response that does not break the loop (https://github.com/home-assistant/core/issues/34776)

## Type of change
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to this library)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: https://github.com/bimmerconnected/bimmer_connected/issues/190, https://github.com/home-assistant/core/issues/34776

## Checklist

- [x] The code change is tested and works locally.
- [x] Tests have been added to verify that the new code works.

### CODE REVIEW

File: bimmer_connected/remote_services.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ class ExecutionState(Enum):
     PENDING = 'PENDING'
     DELIVERED = 'DELIVERED'
     EXECUTED = 'EXECUTED'
+    UNKOWN = 'UNKOWN'
```
Comment (gerard33) [inline bimmer_connected/remote_services.py:36]: Missing a N in UNKNOWN
UNKNOWN = 'UNKNOWN'

File: bimmer_connected/remote_services.py [hunk 2]
```diff
@@ -116,7 +117,7 @@ class RemoteServiceStatus:  # pylint: disable=too-few-public-methods
     def __init__(self, response: dict):
         """Construct a new object from a dict."""
         status = response['executionStatus']
-        self.state = ExecutionState(status['status'])
+        self.state = ExecutionState(status.get('status', 'UNKOWN'))
```
Comment (gerard33) [inline bimmer_connected/remote_services.py:120]: Missing a N in UNKNOWN

File: bimmer_connected/remote_services.py [hunk 3]
```diff
@@ -212,7 +213,7 @@ def _block_until_done(self, service: _Services) -> RemoteServiceStatus:
         while True:
             status = self._get_remote_service_status(service)
             _LOGGER.debug('current state if remote service is: %s', status.state.value)
-            if status.state not in [ExecutionState.PENDING, ExecutionState.DELIVERED]:
+            if status.state not in [ExecutionState.UNKOWN, ExecutionState.PENDING, ExecutionState.DELIVERED]:
```
Comment (gerard33) [inline bimmer_connected/remote_services.py:216]: Missing a N in UNKNOWN

File: test/responses/G31_NBTevo/flash_unkown.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (gerard33) [inline test/responses/G31_NBTevo/flash_unkown.json:1]: Missing a N in unknnown in filename

File: test/test_remote_services.py [hunk 5]
```diff
@@ -9,6 +9,7 @@
 from bimmer_connected.remote_services import RemoteServiceStatus, ExecutionState, PointOfInterest, Message
 from bimmer_connected import remote_services
 
+_RESPONSE_UNKOWN = 'G31_NBTevo/flash_unkown.json'
```
Comment (gerard33) [inline test/test_remote_services.py:12]: Missing a N in unknown

File: test/test_remote_services.py [hunk 6]
```diff
@@ -29,6 +30,9 @@ def test_parse_timestamp(self):
 
     def test_states(self):
         """Test parsing the different response types."""
+        rss = RemoteServiceStatus(load_response_json(_RESPONSE_UNKOWN))
+        self.assertEqual(ExecutionState.UNKOWN, rss.state)
```
Comment (gerard33) [inline test/test_remote_services.py:34]: Missing a N in UNKNOWN (2 times)

File: test/test_remote_services.py [hunk 7]
```diff
@@ -70,6 +74,7 @@ def test_trigger_remote_services(self):
                 r'https://.+/webapi/v1/user/vehicles/{vin}/status'.format(
                     vin=G31_VIN),
                 data_files=[
+                    _RESPONSE_UNKOWN,
```
Comment (gerard33) [inline test/test_remote_services.py:77]: Well you get the point 😉

**Review by gerard33 — CHANGES_REQUESTED** [review]

**Comment by rikroe** [comment]
Well, at least I was VERY consistent 🤣 Thanks!

**Review by gerard33 — APPROVED** [review]

**Pull request merged**